### ISSUE TITLE
How long does it take to receive the dataset email?

### ISSUE THREAD

**Issue by chester256** [ScanNet/ScanNet#36]
3 days has passed.

**Comment by JerryXu0907** [comment]
It took me about a week?

**Comment by chester256** [comment]
I have already got it. Thanks
________________________________
发件人: Zhengjie Xu <notifications@github.com>
发送时间: 2019年9月6日 10:35
收件人: ScanNet/ScanNet <ScanNet@noreply.github.com>
抄送: Zizheng Yan (115010256) <zizhengyan@link.cuhk.edu.cn>; Author <author@noreply.github.com>
主题: Re: [ScanNet/ScanNet] How long does it take to receive the dataset email? (#36)


It took me about a week?

―

**Comment by lapetite123** [comment]
can you download the datasets?  when  I run the script, it occurs "urllib.error.URLError: <urlopen error [Errno 110] Connection timed out>"

**Comment by chester256** [comment]
Yes I can. Can you ping the url specified in the download script? It seems to be a network error.
________________________________
发件人: lapetite123 <notifications@github.com>
发送时间: 2019年9月18日 11:10
收件人: ScanNet/ScanNet <ScanNet@noreply.github.com>
抄送: Zizheng Yan (115010256) <zizhengyan@link.cuhk.edu.cn>; Author <author@noreply.github.com>
主题: Re: [ScanNet/ScanNet] How long does it take to receive the dataset email? (#36)


can you download the datasets? when I run the script, it occurs "urllib.error.URLError: <urlopen error [Errno 110] Connection timed out>"

―

**Comment by lapetite123** [comment]
can anyone provide me with  a database download scripts? It's been five days, I don't receive the dataset email

**Comment by chester256** [comment]
> can anyone provide me with a database download scripts? It's been five days, I don't receive the dataset email

Cloud you please send email to the dataset author ([Angela Dai](https://angeladai.github.io/index.html))?

**Comment by lapetite123** [comment]
I have sent email to the author, but I did't hear from the author for a long time

**Comment by longmalongma** [comment]
can you send me its link?I can not successfully send mail to it.my email is that:1442342449@qq.com

**Comment by rahullabs** [comment]
@angeladai, Hi I have submitted the TOS but till now I didn't get any information about the dataset. Can anyone guide what should I do?
